content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -354,6 +360,23 @@ For example:
 
 The response would look the same as for non dry run request, but the values of some generated fields may differ.
 
+#### Dry Run Authorization
+
+Write authroization is required for the modifying verb for dry-run access.
+
+(`POST`, `PUT`, `PATCH`, and `DELETE`)
+
+This example of an RBAC authorization rule allows dry-run for the verbs
```

Comment:
this is confusing, since this RBAC rule allows much more than dry run

Reply:
Reduced the example to show dry-run authorization to PATCH Deployments